lang/build.gradle

```diff
@@ -62,16 +62,70 @@ tasks.register('javadocJar', Jar) {
 
 publishing {
     publications {
-        create('maven', MavenPublication) {
-            artifactId = 'partiql-lang-kotlin'
+        maven(MavenPublication) {
+            // group id and version are set in the root build.gradle for all projects.
+            artifactId = "partiql-lang-kotlin"
+
+            from components.java
+            artifact sourcesJar
```

Comment:
Think you have to configure both this artifacts as gradle tasks, `javadocJar` specially since you need to configure `dokka` to generate javadoc based on the kdoc, example: https://github.com/amzn/ion-schema-kotlin/blob/master/build.gradle#L53

Reply:
Next commit will have `dokka` and the `javadocJar` tasks configured.  I inspected the resulting jars which appear fine AFAIK.